go/vt/sqlparser/sql.go

```diff
@@ -2209,154 +1945,290 @@ var yyAct = [...]int{
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	0, 627, 0, 0, 0, 0, 0, 0, 0, 0,
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	0, 0, 0, 137, 0, 0, 0, 0, 0, 0,
+	258, 0, 0, 0, 202, 0, 233, 140, 154, 115,
+	101, 111, 0, 139, 180, 209, 213, 0, 0, 0,
+	0, 0, 123, 0, 211, 190, 249, 0, 0, 192,
+	210, 158, 239, 203, 248, 259, 260, 236, 256, 264,
+	226, 104, 235, 247, 120, 221, 106, 245, 232, 169,
+	149, 150, 105, 0, 207, 128, 135, 125, 182, 242,
+	243, 124, 267, 112, 255, 108, 113, 254, 176, 238,
+	246, 170, 163, 107, 244, 168, 162, 153, 132, 142,
+	200, 160, 201, 143, 173, 172, 174, 0, 0, 0,
+	230, 252, 268, 117, 0, 0, 237, 262, 263, 0,
+	0, 118, 136, 131, 199, 175, 114, 145, 227, 152,
+	159, 206, 266, 189, 212, 121, 251, 228, 0, 0,
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	0, 0, 0, 0, 0, 0, 0, 100, 109, 156,
+	265, 204, 134, 253, 0, 0, 127, 0, 0, 0,
 	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 102,
+	103, 110, 116, 122, 126, 130, 133, 138, 141, 144,
+	146, 147, 148, 151, 161, 164, 165, 166, 167, 177,
+	178, 179, 181, 184, 185, 186, 187, 188, 191, 193,
+	194, 195, 196, 197, 198, 205, 208, 214, 215, 216,
+	217, 218, 219, 220, 222, 223, 224, 225, 231, 234,
+	240, 241, 250, 257, 261, 26, 0, 0, 0, 0,
+	0, 0, 0, 0, 0, 0, 0, 183, 0, 0,
+	0, 0, 0, 0, 0, 0, 129, 0, 0, 0,
+	0, 155, 0, 0, 0, 0, 157, 0, 0, 229,
+	171, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	0, 0, 0, 0, 0, 61, 0, 0, 412, 0,
+	0, 0, 0, 0, 0, 0, 0, 119, 0, 0,
 	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-	0, 0, 0, 0, 137, 0, 0, 0, 0, 0,
-	0, 258, 0, 0, 0, 202, 0, 233, 140, 154,
-	115, 101, 111, 0, 139, 180, 209, 213, 0, 0,
-	0, 0, 0, 123, 0, 211, 190, 249, 0, 0,
-	192, 210, 158, 239, 203, 248, 259, 260, 236, 256,
-	264, 226, 104, 235, 247, 120, 221, 106, 245, 232,
-	169, 149, 150, 105, 0, 207, 128, 135, 125, 182,
-	242, 243, 124, 267, 112, 255, 108, 113, 254, 176,
-	238, 246, 170, 163, 107, 244, 168, 162, 153, 132,
-	142, 200, 160, 201, 143, 173, 172, 174, 0, 0,
-	0, 230, 252, 268, 117, 0, 0, 237, 262, 263,
-	0, 0, 118, 136, 131, 199, 175, 114, 145, 227,
-	152, 159, 206, 266, 189, 212, 121, 251, 228, 0,
 	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-	0, 0, 0, 0, 0, 0, 0, 0, 100, 109,
-	156, 265, 204, 134, 253, 0, 0, 127, 0, 0,
 	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 	0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-	102, 103, 110, 116, 122, 126, 130, 133, 138, 141,
-	144, 146, 147, 148, 151, 161, 164, 165, 166, 167,
-	177, 178, 179, 181, 184, 185, 186, 187, 188, 191,
-	193, 194, 195, 196, 197, 198, 205, 208, 214, 215,
-	216, 217, 218, 219, 220, 222, 223, 224, 225, 231,
-	234, 240, 241, 250, 257, 261, 183, 0, 0, 0,
-	0, 0, 0, 0, 0, 129, 0, 0, 0, 0,
-	155, 0, 0, 0, 0, 157, 0, 0, 229, 171,
+	0, 0, 137, 0, 0, 0, 0, 0, 0, 258,
+	0, 0, 0, 202, 0, 233, 140, 154, 115, 101,
+	111, 0, 139, 180, 209, 213, 0, 0, 0, 0,
+	0, 123, 0, 211, 190, 249, 0, 0, 192, 210,
+	158, 239, 203, 248, 259, 260, 236, 256, 264, 226,
+	104, 235, 247, 120, 221, 106, 245, 232, 169, 149,
+	150, 105, 0, 207, 128, 135, 125, 182, 242, 243,
```

Comment:
Shouldn't this be:

150, 105, 0, 73, 185, 135, 125, 182, 242, 243,

Reply:
Fixed